Core/src/org/sleuthkit/autopsy/datamodel/FileNode.java

```diff
@@ -147,6 +148,7 @@ private void setIcon(AbstractFile file) {
     public Action[] getActions(boolean context) {
         List<Action> actionsList = new ArrayList<>();
         actionsList.addAll(Arrays.asList(super.getActions(true)));
+        actionsList.add(new FileSystemDetailsAction(content));
         if (!this.getDirectoryBrowseMode()) {
```

Comment:
should this move into some common place such as AbstractFsContentNode, which is the supe type of all three of these node classes?

Reply:
I thought about that but currently we don't have an implementation of the getActions method there so we would need to do the following and I wasn't confident creating yet another array list of actions was preferable to having one like repeated three times.  If having the additional list created is indeed preferable then I will make those changes.
 
List<Action> actionsList = new ArrayList<>();
actionsList.addAll(Arrays.asList(super.getActions(true)));
actionsList.add(new FileSystemDetailsAction(content));
return actionsList;